docs/milestone-1-validation.md

```diff
@@ -0,0 +1,99 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault (check [How to Run Vault](https://github.com/scality/Vault/blob/development/7.10/TESTING.md#how-to-run-1))
```

Comment:
Same as below, linking to a specific file in a git repo feels wrong considering how often our branches change (@tcarmet deletes lots of branches)

Reply:
Same as I explained above, milestone-1 is releasing with 7.10. So for this document, this should be fine